huntsman/scheduler/observation.py

```diff
@@ -79,3 +81,35 @@ def add_field(self, new_field, new_exptime):
 
     def __str__(self):
         return "DitheredObservation: {}: {}".format(self._field, self._exptime)
+
+
+class DitheredFlatObservation(DitheredObservation):
+    """ A DitheredObservation specifically for flat fields."""
+
+    def __init__(self, position, pattern=dither.dice9, n_positions=9,
+                 dither_offset=0.5*u.arcmin, random_offset=0.5*u.arcmin,
+                 *args, **kwargs):
+        """
+        Args:
+            position (str): Center of field, can be anything accepted by
+                `~astropy.coordinates.SkyCoord`.
+        """
+        # Create the observation
+        field = Field('Flat Field', position.to_string('hmsdms'))
+        super().__init__(field=field, *args, **kwargs)
+
+        # Listify the exposure time
+        self.exptime = [self.exptime for _ in range(n_positions)]
+
+        # Setup the dither fields
+        dither_coords = dither.get_dither_positions(
+                field.coord, n_positions=n_positions, pattern=pattern,
+                pattern_offset=dither_offset, random_offset=random_offset)
+        self.field = [Field(f'Dither{i:02d}', c) for i, c in enumerate(dither_coords)]
```

Comment:
```suggestion
        self.field = [Field(f'FlatDither{i:02d}', c) for i, c in enumerate(dither_coords)]
```

Reply:
Will this change change behaviour, or is it just for clear naming?

Reply:
Just for clear naming.